packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -252,20 +252,38 @@ class Viewport implements IViewport {
     vec3.scaleAndAdd(center, center, jVector, (size[1] / 2.0) * spacing[1]);
     vec3.scaleAndAdd(center, center, kVector, (size[2] / 2.0) * spacing[2]);
 
+    const imageToWorld: mat4 = [
```

Comment:
This is not imageToWorld (since it does not have Origin in the 4th column). Maybe call it `directionMatrix`?